zio-http/src/main/scala/zhttp/http/Cookie.scala

```diff
@@ -120,6 +140,30 @@ final case class Cookie(
     cookie.flatten.mkString("; ")
   }
 
+  /**
+   * Signs cookie content with a secret
+   */
+  private def signContent(secret: String): String = {
```

Comment:
`sign` should only return the signature not combine it with the original content.